ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v1/fileinput/clientlibs/site/js/fileinputwidget.js

```diff
@@ -460,5 +461,11 @@ if (typeof window.FileInputWidget === 'undefined') {
                 }
             }
         }
+
+        #isFileDuplicate(currFile) {
```

Comment:
This would just not attach the file, ideally, we need to update the user with some message saying why was the file not added, like we do for file size, invalid file name and mime type.